aptos-move/mvhashmap/src/versioned_group_data.rs

```diff
@@ -355,6 +355,7 @@ impl<T: Hash + Clone + Debug + Eq + Serialize, V: TransactionWrite> VersionedGro
                     })
             })
             .collect::<Result<Vec<_>, MVGroupError>>()?;
+        // TODO[agg_v2]: we should be able to propagate the error out?
```

Comment:
you mean change the error type of this function from MVGroupError to PartialVMError?

Reply:
We cannot because we have MVGroupError, but I fixed it turning  `TagSerializationError` into `TagSerializationError(PartialVMError)`